simplified-ui-images/src/main/java/org/nypl/simplified/ui/images/ImageAccountIcons.kt

```diff
@@ -31,28 +27,13 @@ object ImageAccountIcons {
     @DrawableRes defaultIcon: Int,
     iconView: ImageView
   ) {
-    val request: RequestCreator
-    val logoURI: URI? = account.logoURI?.hrefURI
-    if (logoURI != null) {
-      LOG.debug("configuring account logo: {}", logoURI)
-      request = loader.load(logoURI.toString())
-    } else {
-      request = loader.load(defaultIcon)
-    }
-
-    request.into(
-      iconView,
-      object : Callback {
-        override fun onSuccess() {
-          iconView.visibility = View.VISIBLE
-        }
-
-        override fun onError(e: Exception) {
-          LOG.error("failed to load account icon: ", e)
-          iconView.setImageResource(defaultIcon)
-          iconView.visibility = View.VISIBLE
-        }
-      }
-    )
+    val uri = account.logoURI?.hrefURI ?: return
+    this.logger.debug("configuring account logo: {}", uri)
```

Comment:
We've got to adjust or eliminate this logging at some point. This log statement in particular ends up creating a lot of entropy in logs that reduces our ability to gzip them for error reports.

Reply:
Yeah, we could probably move a lot of these to trace level logs. Is that what you were thinking?

Reply:
Yeah, something along those lines.